content/integrations/azure.md

```diff
@@ -51,6 +59,8 @@ Related integrations include:
 Integrating Datadog with Microsoft Azure can be done via the Azure Command Line Interface or through the Azure portal 
```

Comment:
Missing a period on this sentence. Also this is passive voice, I'd rather our docs sound more active where natural, e.g:

"You can integrate your Microsoft Azure account with Datadog using the Azure CLI tool or the Azure portal."